content/id/docs/tasks/debug-application-cluster/resource-usage-monitoring.md

```diff
@@ -0,0 +1,57 @@
+---
+content_type: concept
+title: Perangkat untuk Memantau Sumber Daya
+---
+
+<!-- overview -->
+
+Untuk penyekalaan aplikasi dan memberikan Service yang handal, kamu perlu
+memahami bagaimana aplikasi berperilaku ketika aplikasi tersebut digelar. Kamu bisa memeriksa
+kinerja aplikasi dalam klaster Kubernetes dengan memeriksa Container,
+[Pod](/id/docs/user-guide/pods), [Service](/id/docs/user-guide/services), dan
```

Comment:
we don't have these yet
```suggestion
[Pod](/docs/user-guide/pods), [Service](/docs/user-guide/services), dan
```

Reply:
Ok. But, I think we need to create this script `update-user-guide-links.py` to handle redirection "user-guide"